telegram/ext/_application.py

```diff
@@ -959,8 +961,17 @@ def run_webhook(
                 header isn't set or it is set to a wrong token.
 
                 .. versionadded:: 20.0
-            unix (:class:`pathlib.Path` | :obj:`str`, optional): Path to the unix socket file. Path
-                does not need to exist, in which case the file will be created.
+            unix (:class:`pathlib.Path` | :obj:`str` | :class:`socket.socket`, optional): Can be
+                either:
+
+                * the path to the unix socket file as :class:`pathlib.Path` or :obj:`str`, which
```

Comment:
I would still like to split the paragraph contents into a few shorter sentences 🙈

Reply:
They are so carefully formulated sentences though

Reply:
At the very least the part in braces can be a standalone sentence and in the second bullet point ", however" → ". However".